release-notes.md

```diff
@@ -4,7 +4,17 @@ We'll be updating this document along with each update that is merged into maste
 
 We are now on Testflight :tada:, please forward us an email at hello@dwyl.io to request beta access
 
-## Version 1.1 (Build 5) _Current_
+## Version 1.1 (Build 7) _Current_
```

Comment:
love that the release notes are expanding. 
would quite like to have screenshots and even GIFs in here ... 🤔